source/about/install-mattermost-for-microsoft-teams-plugin.rst

```diff
@@ -156,7 +156,15 @@ See our `collaborate using the Microsoft Teams plugin </collaborate/collaborate-
 Trobleshooting
 --------------
 
-If you encounter issues when connecting user accounts or linking channels, restart the plugin as a Mattermost system admin. 
+FAQ
+~~~~~~~~~~~~~~
+**Can I connect my Mattermost account to a Microsoft Teams account of a different email address?**
+
+No. Currently, only accounts with the same email addresses are allowed to be connected.
+
+**I'm experiencing multiple issues connecting user accounts and linking channels.**
```

Comment:
I think we should deprecate this advice -- it stems from the era where the plugin would crash and stop working altogether, but that's not relevant anymore. The specific advice here sounds too much like "Restart windows" :D